docker-compose.yml

```diff
@@ -14,16 +14,20 @@ services:
   # Jaeger
   jaeger:
     image: jaegertracing/all-in-one
-    container_name: jaeger
     ports:
       - "16686:16686"                    # Jaeger UI
       - "14250"                          # Jaeger model.proto endpoint
+      - "16685:16685"
     logging: *logging
+    healthcheck:
+      test: ["CMD", "wget", "--spider", "localhost:16686"]
+      interval: 1s
+      timeout: 3s
+      retries: 60
 
   # Collector
   otelcol:
     image: otel/opentelemetry-collector-contrib:0.56.0
-    container_name: otel-col
```

Comment:
Having container names being part of each of the services can cause colliding issues with other applications running a the same time. I believe it is better to leave docker-compose to take care of the name.